### PR TITLE
enable passport page

### PR DESCRIPTION
**Pull request by endertunc**
This PR enables the `Passport` page. It will allow people to fill their passport with basic informations such as education, experiences, skills, achievements etc.

We will update the confluence page to give more insight about the process. Additional features are still in progress.

### CODE REVIEW

File: backend/src/test/scala/com/lunatech/cc/api/PeopleServiceSpec.scala [hunk 1]
```diff
@@ -2,6 +2,7 @@
 //
 //import com.lunatech.cc.api.Data._
 //import com.lunatech.cc.api.services.ApiPeopleService
+//import com.lunatech.cc.api.services.PeopleService.Config
```
Comment (nMoncho) [inline backend/src/test/scala/com/lunatech/cc/api/PeopleServiceSpec.scala]: Should we remove this file entirely?

File: backend/src/main/scala/com/lunatech/cc/api/CompetenceCenterApi.scala [hunk 2]
```diff
@@ -84,8 +87,17 @@ object CompetenceCenterApi extends App {
   }
 
   // For endpoints that require an API key OR a Google authenticated user.
+  private val staticTokenVerifier = new StaticTokenVerifier("erik.janssen@lunatech.nl")
```
[the remark this answers is not in the document]
Comment (endertunc) [reply backend/src/main/scala/com/lunatech/cc/api/CompetenceCenterApi.scala]: No, it does not have to. It's just a place holder.

File: backend/src/main/resources/application.conf [hunk 3]
```diff
@@ -41,6 +41,10 @@ services {
     name = "people.lunatech.com:80"
     api-key = ${PEOPLE_API_KEY}
   }
+  skillmatrix {
+    name = "techmatrix.lunatech.com:80"
+    api-key = ${TECHMATRIX_API_KEY}
```
Comment (endertunc) [inline backend/src/main/resources/application.conf]: I am not sure how the application is deployed but we need to provide TECHMATRIX_API_KEY env variable.

Comment (nMoncho) [reply backend/src/main/resources/application.conf]: Yes, it's not a problem, CleverCloud let us defined env vars, PEOPLE_API_KEY is the same

**Pull request merged**